locobot/agent/locobot_mover.py

```diff
@@ -155,8 +159,24 @@ def look_at(self, obj_pos, yaw_deg, pitch_deg):
         return "finished"
 
     def point_at(self, target_pos):
+        """Executes pointing the arm at the specified target pos. 
+
+        The robot will try to move to within a certain distance of the target position and then point to it.
+
+        Args:
+            target_pos ([x,y,z]): canonical coordinates to point to.
+
+        Returns:
+            string "finished"
+        """
+        # Part 1: move to within a certain distance of the target.
         pos = self.get_base_pos_in_canonical_coords()
         yaw_rad, pitch_rad = get_camera_angles([pos[0], ARM_HEIGHT, pos[1]], target_pos)
+        self.move_absolute([get_move_target_for_point(pos, target_pos, yaw_rad)])       
```

Comment:
imo this should be in tasks.py:  a Point task should first place a Move task, instead of everything being in a blob in the mover

Reply:
Added a move task inside point.